src/regressionmodel.jl

```diff
@@ -12,6 +12,15 @@ is specified via `typical`. Note that this is also applied to categorical contra
 thus yielding an average of the contrast, weighted by the balance of levels in the data
 set used to fit the model.
 
+`typical` can either be a single, scalar-valued function or a dictionary
+matching term symbols to scalar-valued functions. The use of a dictionary
+allows specifying different `typical` functions for different input variables.
+In this case, `typical` functions must be provided for all term variables
+except the intercept. If there is a single term that should be "typified"
+differently than others, then the use of `DataStructures.DefaultDict` may
+be useful to create a default `typical` value with only the exception
+explicitly specified.
```

Comment:
```suggestion
explicitly specified. For example:

```julia
using DataStructures
typical = DefaultDict(() -> mean)  # default to x -> mean(x)
typical[:sex] = v -> 0.0                   # typical value for :sex
```
```

(Not sure how to escape triple quotes inside a github suggestion, the closing quotes get dropped...)

Reply:
just use more quotes ;) 

`````

````
```julia
using Stuff
```
````

`````

Reply:
(that's quintuple backquotes around the whole thing)